platforms/hyperledger-fabric/configuration/samples/network-fabric-add-new-channel.yaml

```diff
@@ -166,7 +166,7 @@ network:
         url: ca.supplychain-net:7054
         certificate: file/server.crt        # This has not been implemented in 0.2.0.0
   
-      cloud_provider: aws   # Options: aws, azure, gcp, minikube
+      cloud_provider: aws   # Options: aws, azure, gcp, digitalocean, minikube
```

Comment:
Add similar change in the docs folder which contains all our readthedocs documentation